AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Main.py

```diff
@@ -7,122 +7,131 @@
         from .tests.duplicate_prefab import DuplicatePrefab_ContainingNestedEntitiesAndNestedPrefabs as test_module
 
     # Instantiate Prefab Tests
 
-    class test_InstantiatePrefab_ContainingASingleEntity(EditorSharedTest):
+    class test_InstantiatePrefab_ContainingASingleEntity(EditorBatchedTest):
         from .tests.instantiate_prefab import InstantiatePrefab_ContainingASingleEntity as test_module
 
-    class test_InstantiatePrefab_FromCreatedPrefabWithSingleEntity(EditorSharedTest):
+    class test_InstantiatePrefab_FromCreatedPrefabWithSingleEntity(EditorBatchedTest):
         from .tests.instantiate_prefab import InstantiatePrefab_FromCreatedPrefabWithSingleEntity as test_module
 
     # Open Level Tests
 
-    class test_OpenLevel_ContainingTwoEntities(EditorSharedTest):
+    class test_OpenLevel_ContainingTwoEntities(EditorBatchedTest):
         from .tests.open_level import OpenLevel_ContainingTwoEntities as test_module
-
-    # Overrides Tests
-
-    class test_EditEntity_UnderImmediateInstance(EditorSharedTest):
-        from .tests.overrides import EditEntity_UnderImmediateInstance as test_module
-
-    class test_EditEntity_UnderNestedInstance(EditorSharedTest):
-        from .tests.overrides import EditEntity_UnderNestedInstance as test_module
         
     # Prefab Notifications Tests
 
-    class test_PrefabNotifications_PropagationNotificationsReceived(EditorSharedTest):
+    class test_PrefabNotifications_PropagationNotificationsReceived(EditorBatchedTest):
         from .tests.prefab_notifications import PrefabNotifications_PropagationNotificationsReceived as test_module
 
-    class test_PrefabNotifications_RootPrefabLoadedNotificationsReceived(EditorSharedTest):
+    class test_PrefabNotifications_RootPrefabLoadedNotificationsReceived(EditorBatchedTest):
         from .tests.prefab_notifications import PrefabNotifications_RootPrefabLoadedNotificationsReceived as test_module
 
     # Reparent Prefab Tests
 
-    class test_ReparentEntity_UnderEntityHierarchies(EditorSharedTest):
+    class test_ReparentEntity_UnderEntityHierarchies(EditorBatchedTest):
         from .tests.reparent_prefab import ReparentEntity_UnderEntityHierarchies as test_module
+
+
+@pytest.mark.SUITE_periodic
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomationOverrides(EditorTestSuite):
+
+    # These tests will execute with prefab overrides enabled
+    EditorTestSuite.global_extra_cmdline_args.append("--regset=O3DE/Preferences/Prefabs/EnableOverridesUx=true")
+
+    # Overrides Tests
+
+    class test_EditEntity_UnderImmediateInstance(EditorBatchedTest):
+        from .tests.overrides import EditEntity_UnderImmediateInstance as test_module
+
+    class test_EditEntity_UnderNestedInstance(EditorBatchedTest):
+        from .tests.overrides import EditEntity_UnderNestedInstance as test_module
```

Comment:
does this need to move to the TestSuite_Periodic.py file or does a Cmakelists.txt entry need to update for this to be discovered as part of SUITE_periodic?

Reply:
Whoops, good catch. This was a copy paste error.

Reply:
Fixed!